scene/resources/dynamic_font.cpp

```diff
@@ -849,6 +864,24 @@ Size2 DynamicFont::get_char_size(CharType p_char, CharType p_next) const {
 	return ret;
 }
 
+String DynamicFont::get_available_chars() const {
+	if (!data_at_size.is_valid())
+		return "";
```

Comment:
```suggestion
	if (!data_at_size.is_valid()) {
		return "";
	}
```

Reply:
Fixed!

In the PR for 3.2 I've kept the braceless style, to match other occurrences of the same check across the file.